habitat-hitl/habitat_hitl/config/hitl_defaults.yaml

```diff
@@ -34,6 +34,10 @@ habitat_hitl:
       code_available: 200  # "Ok"
       code_unavailable: 503  # "Service Unavailable"
 
+    active_features:
```

Comment:
I left a similar comment in an earlier PR. "Active" means nothing and "features" is too broad. Maybe your group here is getting at what kinds of data are sent and received; maybe a group name along those lines.